ml-proto/test/left-to-right.wast

```diff
@@ -1,20 +1,22 @@
 (module
-  (memory 1)
+  (memory 9)
   
   ;; The idea is: We reset the memory, then the arithmetic instruction calls $*_left and $*_right.
   ;; $*_left stores the number 1, and $*_right stores the number 2 (both at address 0)
   ;; Then we read the value at address 0. If it's 0, the VM did an unwanted optimzation.
```

Comment:
I needed some space for the store operations to store their data in. I use the first 8 bytes to do so and use the 9th one to store the order information.